firebase-vscode/src/data-connect/terminal.ts

```diff
@@ -54,15 +54,15 @@ export function runTerminalTask(
         }
       }
     });
-    const task = await vscode.tasks.executeTask(
-      new vscode.Task(
-        { type },
-        vscode.TaskScope.Workspace,
-        taskName,
-        "firebase",
-        new vscode.ShellExecution(command, executionOptions),
-      ),
+    const task = new vscode.Task(
+      { type },
+      vscode.TaskScope.Workspace,
+      taskName,
+      "firebase",
+      new vscode.ShellExecution(command, executionOptions),
     );
+    task.presentationOptions = { focus: true };
```

Comment:
On a second thought, maybe we don't need focus for running the emulators? That command almost never prompts for anything interactively. Focusing the terminal for init and deploy makes sense though

Reply:
I agree, we can add it to the call signature

Reply:
Done